components/Controllers/AuctionPageContent/AuctionGraph.tsx

```diff
@@ -206,22 +207,18 @@ export function AuctionGraph({
           )}`
         : 'SOLD: ......';
     }
-  }, [
-    auctionUnderlyingPrice,
-    controller.underlying.decimals,
-    auctionCompleted,
-  ]);
+  }, [auctionUnderlyingPrice, controller.underlying, auctionCompleted]);
 
   const floorLabel = useMemo(() => {
     const floorPrice =
-      oracleInfo[auction.auctionAssetContract.id].price.toFixed(2);
+      oracleInfo[auction.auctionAssetContract.id].price.toFixed(4);
     const floorPricePadding = Array(
-      floorEthPrice.toString().length + 4 - (floorPrice.toString().length + 1),
+      floorPrice.toString().length + 5 - (floorUSDPrice.toString().length + 1),
     )
       .fill('\t')
       .join('');
-    return `\n\n\n\n${floorEthPrice} ETH\n${floorPricePadding}$${floorPrice}`;
-  }, [oracleInfo, floorEthPrice, auction.auctionAssetContract.id]);
+    return `\n\n\n\n${floorPrice} WETH\n${floorPricePadding}$${floorUSDPrice}`;
```

Comment:
I'd love it if we had a more readable way to do this text, but I guess we're limited by the chart library?

Reply:
yeah unfortunately since we're using a very limited plugin to the chart library there isn't a nice way to do this afaik